src/components/Chat/ActionChat.js

```diff
@@ -50,7 +50,7 @@ const styles = StyleSheet.create({
     position: 'absolute',
     bottom: -20,
     right: 20,
-    backgroundColor: COLORS.lightgrey,
+    backgroundColor: COLORS.gray100,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.lightgrey` to `COLORS.gray100` for the background color of the absolute positioned element seems fine as long as it fits into the overall design scheme and doesn't affect readability or usability.
```diff
-    backgroundColor: COLORS.lightgrey,
+    backgroundColor: COLORS.gray100,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->